.github/workflows/build.yaml

```diff
@@ -57,6 +59,20 @@ jobs:
         run: |
           SMQ_ES_TYPE=es_redis make mqtt
 
-      - name: Build and push Dockers
+      - name: Build and push Docker images
         run: |
-          make latest -j $(nproc)
+          make latest -j $(nproc) # This should build and push docker.io/absmach/supermq:latest
+
+      - name: Notify amdm Helm Chart Repository of New Image
+        if: success() && github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.REPO_DISPATCH_PAT_TO_AMDM }}
+          repository: absmach/amdm
+          event-type: supermq-image-updated
+          client-payload: |
+            {
+              "image_repository": "docker.io/absmach/supermq",
```

Comment:
I don't think this is repo name , it should end with service name we should do it for all the service